contributing/code/standards.rst

```diff
@@ -162,6 +147,8 @@ Structure
 * Use ``return null;`` when a function explicitly returns ``null`` values and
   use ``return;`` when the function returns ``void`` values;
 
+* Do not add the ``void`` return type to methods;
```

Comment:
I think we only recommend this for tests.

Reply:
Yes

Reply:
I was under the impressions that we did this for all methods, as `void` doesn't really add anything and this way, we keep a door open to make the method return something (without the need for an advanced BC layer).